gz-token.html

```diff
@@ -22,145 +22,93 @@
       id='ajax'
       url=''
       handle-as='json'
-      on-response='hresponse'
-      on-error="herror"
+      on-response='_onAjaxResponse'
+      on-error="_onAjaxError"
       debounce-duration='300'>
     </iron-ajax>
 
     <div id="config"  hidden$=[[!config]]>
 
-      <div><h1>{{title}}</h1></div>
-      <div><h4>{{caption}}</h4></div>
+      <div><h1>[[title]]</h1></div>
 
-      <div id="feedback" style="display:none">
-        <span id="feedback_text"/>
-      </div>
-
-      <div>
-        Auth server URL:<input id=url value="{{url::input}}"></input>
-      </div>
-      <div>
-        Data (JSON):<input id=data value="{{data::input}}"></input>
-      </div>
       <div>
-        <button on-tap="tapToken" id="buttonToken">token</button>
+        Auth server URL:<input value="{{url::input}}"></input>
       </div>
       <div>
-        Token: {{token}}
+        Token: [[token]]
       </div>
 
       <div>
-        <button on-tap="tapLogin" id="buttonLogin">login</button>
-        <button on-tap="tapLogout" id="buttonLogout">logout</button>
+        <button on-tap="tapLogin">Login</button>
+        <button on-tap="tapLogout">Logout</button>
       </div>
 
     </div>
 
   </template>
 
-  <script>
+  <!-- AUTH0 -->
+  <script src="//cdn.auth0.com/js/lock/10.1.0/lock.min.js"></script>
 
+  <script>
   (function() {
     'use strict';
 
     Polymer({
       is: 'gz-token',
 
-      tapLogin: function() {
-        this.state = 'Requesting login'
-        let url = this.url
-        if (!url.endsWith('/'))
-          url += '/'
-        url += 'login'
-        this.$.ajax.url = url
-        this.$.ajax.withCredentials = true
-        this.$.ajax.generateRequest()
-      },
+      /// Called when widget is attached.
+      attached: function() {
+        const that = this
 
-      tapLogout: function() {
-        // Clear for refresh
-        window.localStorage.removeItem("username");
-        window.localStorage.removeItem("token");
+        const auth0_id = getConfig().auth0_id
```

Comment:
This breaks https://osrf.github.io/gz-token/components/gz-token/demo
please use properties instead of getConfig()